frappe/public/js/frappe/list/base_list.js

```diff
@@ -417,7 +417,13 @@ frappe.views.BaseList = class BaseList {
 	get_group_by() {
 		let name_field = this.fields && this.fields.find((f) => f[0] == "name");
 		if (name_field) {
-			return frappe.model.get_full_column_name(name_field[0], name_field[1]);
+			return frappe.model
+				.get_full_column_name(name_field[0], name_field[1])
+				.concat(
+					Object.entries(this.link_field_title_fields || {}).map(
+						(entry) => " ," + entry.join("_")
+					)
+				);
```

Comment:
This should be done server side instead? 

db_query implicitly joins things, so it should be responsible for making joins work on postgres.

Reply:
Actually this pr is a simplification for this [pr](https://github.com/frappe/frappe/pull/22092), i thought it would be eiasier to include the field in the group by as it is done with 'name', instead of doing extra work in the backend.